tests/test_resources/test_clusters/test_on_demand_cluster.py

```diff
@@ -0,0 +1,21 @@
+import tests.test_resources.test_clusters.test_cluster
+
+
+class TestOnDemandCluster(tests.test_resources.test_clusters.test_cluster.TestCluster):
+
+    MAP_FIXTURES = {"resource": "cluster"}
+
+    UNIT = {"cluster": []}
+    LOCAL = {"cluster": []}
+    MINIMAL = {"cluster": ["ondemand_cpu_cluster"]}
+    THOROUGH = {"cluster": ["ondemand_cpu_cluster", "ondemand_https_cluster_with_auth"]}
+    MAXIMAL = {
+        "cluster": [
```

Comment:
should we add a sagemaker cluster here? or first wait to migrate that test suite into the new format